pyfarm/agent/sysinfo/network.py

```diff
@@ -120,15 +130,12 @@ def hostname(trust_name_from_ips=True):
     # produce the wrong information when resolving the local
     # hostname.
     if len(reverse_hostnames) == 1 and trust_name_from_ips:
-        return reverse_hostnames.pop()
+        return reverse_hostnames.pop().lower()
```

Comment:
Do we still need the lower here since the hostname should have been inserted after calling `.lower()`?  If yes, we probably are missing a `.lower()` call somewhere.